### DISCUSSION TITLE
Ability to hide elements

### DISCUSSION THREAD

**Discussion by dvXcode** [Azure/api-management-developer-portal#2543] [Feature request]
## Expected behavior

Right now, user can only delete elements. It will be nice if it was possible to hide element with help of extra button over selected element.

## Is your portal managed or self-hosted?

Managed

![HideSelectedElementDP](https://user-images.githubusercontent.com/51912361/83136247-7344ce00-a0e7-11ea-8bf4-5dfafc30feaa.png)

**Comment by mikebudzynski** [comment]
@dvXcode Can you please elaborate on the scenario here? Is it for testing purposes?
